cmd/trace-agent/test/testsuite/otlp_test.go

```diff
@@ -19,7 +19,7 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	"go.opentelemetry.io/collector/model/otlpgrpc"
-	"go.opentelemetry.io/collector/model/pdata"
+	"go.opentelemetry.io/collector/pdata/ptrace"
```

Comment:
Is it worth updating `pkg/trace/go.mod` too?

Reply:
Not sure I understand what you mean, you would prefer to stay on the old version for `pkg/trace` ?

Reply:
No, I meant upgrading the `pkg/trace/go.mod` file too, not just the root `go.mod` file, so that they both use the newer (latest) version.

Reply:
Whoops, seems like you already did that. Resolving. My bad.